steps/create_new_branch.yml

```diff
@@ -0,0 +1,24 @@
+id: create_new_branch
```

Comment:
I think the step is missing from the quest.yaml

Reply:
added